game-core/src/main/java/games/strategy/triplea/delegate/FinishedBattle.java

```diff
@@ -30,7 +30,6 @@
  */
 public class FinishedBattle extends AbstractBattle {
   private static final long serialVersionUID = -5852495231826940879L;
-  private final Set<Territory> m_attackingFrom = new HashSet<>();
```

Comment:
Are you sure this is gonna be save compatible?

Reply:
Good Point.
This depends on whether or not this object is being sent over the network.
If it's not, this change would be perfectly fine and user would just not be able to downgrade with their savegame.